packages/auth/src/features/authenticate/service.ts

```diff
@@ -211,3 +216,21 @@ export function verifyToken(token: string): ITokenPayload {
 export function getPublicKey() {
   return publicCert
 }
+
+export async function postUserActionToMetrics(
+  action: string,
+  practitionerId: string,
+  token: string
+) {
+  const url = resolve(METRICS_URL, '/audit/events')
+  const body = { practitionerId: practitionerId, action: action }
```

Comment:
The influxDB has it as part of the requirement to save practitionerId

Reply:
That's fine and yea, we should still store practitionerId. I'd just make it so that it would be the `metrics` service that fetches the practitionerId based on the user id of whoever is making the request